projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-column.model.ts

```diff
@@ -96,18 +96,23 @@ export class TableWidgetColumnModel {
       id: this.value.resultAlias(),
       name: this.value.name,
       display: this.display,
-      title: this.title,
+      title: this.title ?? this.value.name,
       titleTooltip: this.titleTooltip,
       alignment: this.alignment,
       width: this.width,
       visible: this.visible,
+      editable: this.isEditable(),
       filterable: this.filterable,
       sort: this.sort,
       onClick: this.buildClickHandlerIfDefined(),
       specification: this.value
     };
   }
 
+  private isEditable(): boolean {
+    return this.value.resultAlias() === this.value.name; // These do not match if composite column or aggregation
```

Comment:
I think what you are saying makes sense for the json defined mode since those columns are already defined and may have aggregations or be composite. This check was for the attribute fetching to column mode, which requires doesn't allow anything beyond attributes (not metrics) at the moment.

Let me add some additional logic to make this check work for either mode. Does that solve this?

Reply:
So all this value means is - can we toggle this column visible or hidden, right? So I would think that should work for any validly defined column no matter the source. Whether we're fetching attributes and generating columns or providing them in json, assuming it's a valid column I'm not sure what would prevent any column from being toggleble.

Reply:
Yep, after looking at the code again I actually was just coming back to comment. Ignore last message. This method is only used for json defined columns, so its always true here. Fix incoming.